ServeEasyApp/db.yaml

```diff
@@ -0,0 +1,4 @@
+mysql_host: 'localhost'
+mysql_user: 'root'
```

Comment:
- [ ] add comment beside the line to change user name
- [ ] also change the password to a common name rather a personal one